hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/bootstrap/HoodieBootstrapSchemaProvider.java

```diff
@@ -48,7 +48,7 @@ public HoodieBootstrapSchemaProvider(HoodieWriteConfig writeConfig) {
   public final Schema getBootstrapSchema(HoodieEngineContext context, List<Pair<String, List<HoodieFileStatus>>> partitions) {
     if (writeConfig.getSchema() != null) {
       // Use schema specified by user if set
-      Schema userSchema = Schema.parse(writeConfig.getSchema());
+      Schema userSchema = new Schema.Parser().parse(writeConfig.getSchema());
```

Comment:
I see one more usage in `AvroFileDeltaInputWriter` in hudi-integ-test. Though it's not critical but would be good to fix there as well.

Reply:
Oops, didn't see it, @slfan1989 , can you fire another minor PR to track this.

Reply:
Thanks for helping to review the code @codope @danny0405 , I will continue to follow up.

Reply:
@codope I will modify it to the mock method.